### DISCUSSION TITLE
Is it me or the current patch for Raspberry pi 4/Raspbian OS 64 bit is not available (bcm43455c0/7_45_241)

### DISCUSSION THREAD

**Discussion by vricosti** [seemoo-lab/nexmon#605] [Q&A]
Hi,
When following some tutorials for patching wifi chipset for raspberry pi, I dont' manage to do it with my rpi4 running raspbian 64bits:

```
pi@raspberrypi:~/Dev/nexmon $ uname -a
Linux raspberrypi 6.1.21-v8+ #1642 SMP PREEMPT Mon Apr  3 17:24:16 BST 2023 aarch64 GNU/Linux
pi@raspberrypi:~/Dev/nexmon $ ethtool -i wlan0
driver: brcmfmac
version: 7.45.241
firmware-version: 01-703fd60
expansion-rom-version: 
bus-info: mmc1:0001:1
supports-statistics: no
supports-test: no
supports-eeprom-access: no
supports-register-dump: no
supports-priv-flags: no
pi@raspberrypi:~/Dev/nexmon $ lsb_release -a
No LSB modules are available.
Distributor ID:	Debian
Description:	Debian GNU/Linux 11 (bullseye)
Release:	11
Codename:	bullseye
```
So inside nexmon/patches/bcm43455c0/7_45_241 there is only one file and 
make backup-firmware/make install-firmware don't exist ...

**Comment by jlinktu** [comment]
We currently don't provide a patch for the firmware version `7.45.241`.
But you can try to go with version [`7.45.206`](https://github.com/seemoo-lab/nexmon/tree/master/patches/bcm43455c0/7_45_206/nexmon) for now, however, be aware that there is a known problem with frame injection that sometimes leads to a crash.
